### PR TITLE
docs: fix styles

### LINKED ISSUE #51: Replace the word `our`
https://github.com/autowarefoundation/autoware-documentation/pull/40#discussion_r822638199
https://developers.google.com/style/person?hl=en

```
~/ghq/github.com/autowarefoundation/autoware-documentation main
❯ ag " our"
docs/contributing/index.md
28:To ensure our community stays open and healthy, we adhere to the [Contributor Covenant](https://www.contributor-covenant.org/), a widely used [code of conduct](https://github.com/autowarefoundation/autoware/blob/main/CODE_OF_CONDUCT.md) adopted by many other communities listed [here](https://www.contributor-covenant.org/adopters/).
52:See our [support guidelines](../help/support-guidelines.md) for the detailed steps.
68:See our [pull request guidelines](pull-request-guidelines/index.md) for the detailed steps.

docs/contributing/pull-request-guidelines/index.md
8:The following is a general example of our pull request workflow.
15:   - Follow our guidelines when you write code, tests, and documentation.
19:   - Follow our [Commit guidelines](commit-guidelines.md) when commit your change.
28:   - The reviewers will review your code following our [review guidelines](review-guidelines.md).
80:See our [commit guidelines](commit-guidelines.md) for the detailed rules.
```

### PR DESCRIPTION
**Pull request by kenji-miyake**
## Description

Resolves #51.

Fixed some styles in accordance with the following guidelines:
- https://developers.google.com/style/person?hl=en
- https://developers.google.com/style/clause-order

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/design/autoware-concepts/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Universe modules are extensions to the Core module that can be provided by t
...
+The interface design is the most essential piece of the microautonomy architecture, which is classified into internal and external interfaces. The component interface is designed for the components in a Universe module to communicate with those in other modules, including the Core module, within Autoware internally. The AD(Autonomous Driving) API, on the other hand, is designed for the applications of Autoware to access the technology components in the Core and Universe modules of Autoware externally. Designing solid interfaces, the microautonomy architecture is made possible with AWF's partners, and at the same time is made feasible for the partners.
```
Comment (maxime-clem) [inline docs/design/autoware-concepts/index.md:21]: This is unrelated to your changes but that last sentence is not clear to me.

Comment (kenji-miyake) [reply docs/design/autoware-concepts/index.md:21]: Yes, actually it was unclear to me as well. Do you have any ideas for alternative sentences?

Comment (kenji-miyake) [reply docs/design/autoware-concepts/index.md:21]: Or can I merge this if I create an issue to update this page?

**Review by maxime-clem — APPROVED** [review]

File: docs/design/autoware-concepts/index.md (hunk 1 above)
Comment (kenji-miyake) [reply docs/design/autoware-concepts/index.md:21]: There was already an issue. #81
I've added a comment.

**Pull request merged**